docs/unittesting.md

```diff
@@ -50,15 +116,15 @@ Remix facilitates users with various types of customizations to test a contract
 
 **1. Custom Compiler Context**
 
-`Solidity Unit Testing` refers `Solidity Compiler` plugin for compiler configurations. One can provide customized inputs for `Compiler`, `EVM Version` & `Enable Optimization` and these will be the configuration settings used for contract compilation before running unit tests.
+`Solidity Unit Testing` refers `Solidity Compiler` plugin for compiler configurations. One can provide customized inputs for `Compiler`, `EVM Version`, `Enable Optimization` & `runs`. This configuration will be used for contract compilation before running unit tests.
```

Comment:
I had to read the paragraph a couple of times before I understood it.  I didn't quite get that you have to choose the settings in the Solidty Compiler and then these would be the settings used in the Unit Testing plugin.  Here's an edit:

`Solidity Unit Testing`refers to the `Solidity Compiler` plugin for compiler configurations. To specify settings for the `Compiler`, `EVM Version`, `Enable Optimization` & `runs` choose these in the `Solidity Compiler` plugin and this configuration will be used in the `Solidity Unit Testing` plugin for contract compilation before running unit tests.

Reply:
updated